docs/source/sim_overview.rst

```diff
@@ -56,55 +54,28 @@ give you at least a basic exploratory idea of the performance of your review:
 	asreview plot myreview.h5
 	asreview plot DIR_WITH_MULTIPLE_SIMULATIONS
 
-	
+For an example of results of a simulation study, see the `Simulation Results <simulation-results.html>`__ page.
+
+
 Simulation options
 ------------------
-The options for simulating are shown on the `CLI <cli.html>`__ page. We will highlight some of the
-more often used options here.
+The options for simulating are shown on the `CLI <cli.html>`__ page. We will highlight some of the more often used options here. When no additional arguments are specified in the ``asreview simulate`` command, default settings are used.
+
+To make your simulations reproducible you can use the ``--seed`` and ``--init_seed`` options. 'Init
+seed' controls the starting set of papers to train the model on, while the 'seed' controls the
+random generator after initialization.
```

Comment:
controls the seed of the random number generation that is used after initialization.